test/desktop/arduino_interface_test.cc

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "arduino/arduino_interface_mock.h"
+/*
```

Comment:
If we don't need these imports, can we remove them instead of commenting them out?

Although it does look like we do need them, since I see the EXPECT_CALL mock calls below.

Reply:
We need it although it seems that Platformio includes them automatically. We can remove the includes or not. How about?

Reply:
If they don't break the build, I would recommend we keep the imports. That way each source file has the imports it needs to function which may be more resilient.